docs/README.rst

```diff
@@ -59,86 +59,94 @@ Controller Configuration Parameters
+|                     |         |          |             | SDN and Pod network                     |                |
++---------------------+---------+----------+-------------+-----------------------------------------+----------------+
+| manage-routes       | boolean | Optional | false       | Indicates if ``k8s-bigip-ctlr`` should  | true, false    |
+|                     |         |          |             | handle OpenShift Route objects.         |                |
+|                     |         |          |             |                                         |                |
+|                     |         |          |             | Only applicable in the OpenShift        |                |
+|                     |         |          |             | environment.                            |                |
++---------------------+---------+----------+-------------+-----------------------------------------+----------------+
+| route-vserver-addr  | string  | Optional | n/a         | Bind address for virtual server for     |                |
+|                     |         |          |             | OpenShift Route objects.                |                |
+|                     |         |          |             |                                         |                |
+|                     |         |          |             | Only applicable in the OpenShift        |                |
+|                     |         |          |             | environment.                            |                |
++---------------------+---------+----------+-------------+-----------------------------------------+----------------+
+| route-label         | string  | Optional | n/a         | Tells the ``k8s-bigip-ctlr`` to only    |                |
+|                     |         |          |             | watch for OpenShift Route objects with  |                |
+|                     |         |          |             | a label named 'f5type' set to the       |                |
+|                     |         |          |             | specified value.                        |                |
+|                     |         |          |             |                                         |                |
+|                     |         |          |             | Only applicable in the OpenShift        |                |
+|                     |         |          |             | environment.                            |                |
++---------------------+---------+----------+-------------+-----------------------------------------+----------------+
 
+.. note::
+
+  Use the ``node-label-selector`` parameter if you only want the controller to manage specific nodes from the cluster.
+  For example, the BIG-IP device may not be able to reach certain nodes, or the BIG-IP device already manages certain
+  nodes. Therefore, the controller should only watch the nodes outside of these contexts (by using a label).
```

Comment:
I think this might be a little clearer for the last sentence:

Therefore, the controller should only watch the nodes that match the environmental constraints (by using a label).